modules/3d/include/opencv2/3d.hpp

```diff
@@ -2684,47 +2684,58 @@ class CV_EXPORTS Octree {
     Octree();
 
     /** @overload
-    * @brief Create an empty Octree and set the maximum depth.
-    *
-    * @param maxDepth The max depth of the Octree. The maxDepth > -1.
-    */
-    explicit Octree(int maxDepth);
+     * @brief Creates an empty Octree with given maximum depth
+     *
+     * @param maxDepth The max depth of the Octree
+     * @param size bounding box size for the Octree
+     * @param origin Initial center coordinate
+     * @param withColors Whether to keep per-point colors or not
+     * @return resulting Octree
+     */
+    static Octree createWithDepth(int maxDepth, double size, const Point3f& origin = { }, bool withColors = false);
```

Comment:
What about bindings?

Reply:
Implemented python bindings for Octree + a simple test